tests/filters/heuristic_filter_test.py

```diff
@@ -139,6 +139,8 @@ def test_is_prefixed_with_dollar_sign(secret, result):
         ('secret = get_secret_key()', True),
         ('secret = request.headers["apikey"]', True),
         ('secret = hunter2', False),
+        ("<%= ENV['CLIENT_ACCESS_KEY_ID'].presence || 'AKIA123456789ABCDEF1' %>", True),
+        ("ENV['CLIENT_ACCESS_KEY_ID'].presence || 'AKIA123456789ABCDEF1'", False),
```

Comment:
Why would this case return `False`?

Reply:
I'm not sure I understand the question. The tool does not catch this case anyways. The first case is the one we're fixing here. The second one is not caught either way. Do you also wanna catch this one?